201-front-door-standard-premium-geo-filtering/main.bicep

```diff
@@ -0,0 +1,212 @@
+@description('The name of the Front Door endpoint to create. This must be globally unique.')
+param endpointName string = 'afd-${uniqueString(resourceGroup().id)}'
+
+@description('The host name that should be used when connecting from Front Door to the origin.')
+param originHostName string
+
+@description('The name of the SKU to use when creating the Front Door profile.')
+@allowed([
+  'Standard_AzureFrontDoor'
+  'Premium_AzureFrontDoor'
+])
+param frontDoorSkuName string = 'Standard_AzureFrontDoor'
+
+@description('The mode that the WAF should be deployed using. In \'Prevention\' mode, the WAF will block requests it detects as malicious. In \'Detection\' mode, the WAF will not block requests and will simply log the request.')
+@allowed([
+  'Detection'
+  'Prevention'
+])
+param wafMode string = 'Prevention'
+
+@description('The list of country codes that are allowed to access the Front Door endpoint.')

```

Comment:
Are allowed values defined by the RP or the developer?  If the latter where do I get the list of codes?

Reply:
Updated to clarify this in the description as well as in the README.md.